pushapkscript/exceptions.py

```diff
@@ -5,3 +5,7 @@
 class SignatureError(ScriptWorkerTaskException):
     def __init__(self, msg):
         super().__init__(msg, exit_code=STATUSES['internal-error'])
+
+
+class ConfigValidationError(Exception):
```

Comment:
Let's make this error inherit from `ScriptWorkerTaskException`. `scriptworker` provides a nicer error handling when used. https://github.com/mozilla-releng/scriptworker/blob/ab5daa9cbac2d357a550455977f734bee03aebd7/scriptworker/constants.py#L21 is probably the best exit code.